<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeWidget</class>
 <widget class="QWidget" name="welcomeWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>欢迎</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhHiddenText</set>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>160</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonsLayout">
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registButton">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>261</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="typeLayout_2">
    <item>
     <layout class="QHBoxLayout" name="usrnameLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>用户ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usr_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pswdLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>密码  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pswd">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="typeLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>用户类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type">
        <item>
         <property name="text">
          <string>普通用户</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>商家</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>welcomeWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
